src/desktop/layer.rs

```diff
@@ -570,16 +579,15 @@ where
     S: Into<Scale<f64>>,
     P: Into<Point<f64, Physical>>,
 {
-    let location = location.into();
-    let scale = scale.into();
-
-    let elements = SpaceElement::<R, WaylandSurfaceRenderElement>::render_elements(
-        layer,
-        location.to_i32_round(),
+    draw_surface_tree(
```

Comment:
Is this intentionally? It now no longer includes the popups

Reply:
good catch. no, not intentional